dealership/vehicles.py

```diff
@@ -1,15 +1,63 @@
+# things to try: make it DRYer by implementing sale_price and purchase_price
+# methods into Vehicle class, rather than repeating in each type of vehicle.
+
 class Vehicle(object):
+    
+    # sale multipliers 
+    
+    car_sale_mult = 1.2
```

Comment:
We can change these constants caps, following the PEP-8 styleguide - https://www.python.org/dev/peps/pep-0008/#constants

Reply:
Good suggestion! I need to pay better attention to that stuff.